_commands/toys/cowsay.md

```diff
@@ -48,7 +48,7 @@ $ cowsay -s hi there
 ##### Break it down
 There are several provided modes which change the appearance of the cow depending on its particular emotional/physical state
 The user can set this state using these flags
-####cowsay [-f cowfile] [`user_string`]
+#### `cowsay` `[-f cowfile]` `[user_string]`
```

Comment:
If you put a newline between line 50 and 51 it'll actually display that command as a header. If you don't want to display it as a header, please remove the 4 `#`s
![screen shot 2016-03-23 at 5 30 27 pm](https://cloud.githubusercontent.com/assets/7095187/14001573/0276c8d2-f11d-11e5-890f-d1c90b2bd4fb.png)